bdns/dns.go

```diff
@@ -475,3 +488,60 @@ func (dnsClient *DNSClientImpl) LookupCAA(ctx context.Context, hostname string)
 	}
 	return CAAs, nil
 }
+
+// logDNSError logs the provided err result from making a query for hostname. If
+// the err is a `dns.ErrId` instance then the raw Base64 URL encoded bytes of
+// the query (and if not-nil, the response) in wire format is logged as well.
+// This function is called from exchangeOne only for the case where an error
+// occurs querying a hostname that indicates a problem between the VA and the
+// resolver.
+func (dnsClient *DNSClientImpl) logDNSError(hostname string, msg, resp *dns.Msg, err error) {
```

Comment:
Not sure I understand why there is a wrapper method here, could `exchangeOne` not call this function directly and avoid the extra boilerplate?

Reply:
You're right, it's not needed. Thanks! Fixed in c9362e8